test/test_helper.rb

```diff
@@ -10,6 +10,9 @@
 inc_dirs.concat(ADDITIONAL_Y2DIRS)
 ENV["Y2DIR"] = inc_dirs.join(":")
 
+ENV["LC_ALL"] = "C"
```

Comment:
Um, I'm not sure if the plain `C` is the best option, I usually rather prefer `en_US.UTF-8` just to possibly avoid potentional UTF-8 issues. (I do not have any real example where it would make troubles, but my feeling is that using the UTF-8 locale is more safe.)

Reply:
ok, will change it.